packages/wdio-spec-reporter/tests/index.test.js

```diff
@@ -167,7 +173,52 @@ describe('SpecReporter', () => {
             }
 
             const runner = Object.assign({}, RUNNER, {
-                config: { hostname: 'ondemand.saucelabs.com' },
+                config: {
+                    hostname: 'ondemand.saucelabs.com',
+                    capabilities: {}
+                },
+                sessionId: 'ba86cbcb70774ef8a0757c1702c3bdf9'
+            })
+            printReporter.printReport(runner)
+
+            expect(printReporter.write).toBeCalledWith(SAUCELABS_REPORT)
+        })
+
+        it('should print link to SauceLabs job details page if run with Sauce Connect (w3c)', () => {
+            printReporter.suiteUids = SUITE_UIDS
+            printReporter.suites = SUITES
+            printReporter.stateCounts = {
+                passed : 4,
+                failed : 1,
+                skipped : 1,
+            }
+
+            const runner = Object.assign({}, RUNNER, {
```

Comment:
I see that runner is defined most of the times with same/similar set of parameters.
Do you think it make sense to define a function for doing it to reduce amount of code?